portal/src/main/webapp/config_service.jsp

```diff
@@ -59,6 +59,7 @@
             "show.oncokb",
             "show.civic",
             "show.genomenexus",
+            "show.genome_nexus.annotation_sources",
```

Comment:
@leexgh Question re consistency. In the above property (`show.genomenexus`) we spell `genomenexus` without underscore. Are there other properties where we use `genome_nexus`? Otherwise maybe better to keep it consistent and make it `show.genomenexus.annotation_sources`?

Reply:
@inodb good point, I'll change to `genomenexus` to be consistent with others